src/test/scala/tech/cryptonomic/conseil/tezos/DataTypesTest.scala

```diff
@@ -197,6 +215,60 @@ import scala.concurrent.ExecutionContext.Implicits.global
 
       result.futureValue.left.get should contain theSameElementsAs List(InvalidAggregationField("invalid"), InvalidAggregationFieldForType("invalid"))
     }
+
+    "correctly validate aggregation when COUNT function is used" in {
```

Comment:
Not sure if this is confirming that the aggregation is valid or if the validate function works, the test phrasing is not too clear for me.
> Correctly validate
seems to say that we're testing that validate performs correctly.

May I suggest "successfully validates aggregation for any dataType when ..."?

Reply:
changed